fusenet/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-jammy AS build
+
+RUN mkdir /home/nethermind
+
+RUN git clone https://github.com/fuseio/nethermind.git /home/nethermind
```

Comment:
You need to use 
```ARG GIT_URL=...
ARG RELEASE=....````
To let auto build scripts working see https://github.com/pokt-foundation/docker-images/blob/master/.github/workflows/watcher.yml#L49

As example you can use https://github.com/pokt-foundation/docker-images/blob/master/erigon/Dockerfile#L22